app/controllers/groups_controller.rb

```diff
@@ -49,9 +47,7 @@ def update
   def destroy
     @group = current_user.groups.find(params[:id])
 
-    if current_user == @group.creator
-      @group.destroy
-    end
+    @group.destroy if current_user == @group.creator
```

Comment:
Thoughts on creating group owners or admins at some point? That feels like a better fit for this privilege (and others) than limiting it to the group creator (e.g., editing or adding a commitment for someone). I realize this is just a simple refactor and discussing this here may be out of scope.

Reply:
Yup, something like that sounds good. Want to open an issue to start that conversation?